tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/MongoDb/DatabaseNamespaceStruct.cs

```diff
@@ -0,0 +1,24 @@
+// <copyright file="DatabaseNamespaceStruct.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.ComponentModel;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.MongoDb
+{
+    /// <summary>
+    /// MongoDB.Driver.DatabaseNamespace interface for duck-typing
+    /// </summary>
+    [DuckCopy]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct DatabaseNamespaceStruct
```

Comment:
We can change the visibility to internal now :)

Reply:
Good call, done in 4220883d48a8ea9a1235d12e118a53679f64ad31